### PR TITLE
Rfc coding style

### PR DESCRIPTION
**Pull request by krocard**
<a href='#crh-start'></a><a href='#crh-data-%7B%22processed%22%3A%20%5B%22https%3A//github.com/01org/parameter-framework/pull/327%23issuecomment-169080057%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/327%23issuecomment-169282279%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/327%23issuecomment-169298390%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/327%23discussion_r49092655%22%2C%20%22https%3A//github.com/01org/parameter-framework/pull/327%23issuecomment-170671203%22%5D%2C%20%22comments%22%3A%20%7B%22General%20Comment%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/01org/parameter-framework/pull/327%23issuecomment-169080057%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22%40dawagner%20do%20you%20prefer%20this%20style%20%3F%22%2C%20%22created_at%22%3A%20%222016-01-05T17%3A52%3A35Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%2C%20%7B%22body%22%3A%20%22%40krocard%20I%20like%20the%20%5C%22binpack%5C%22%20part%20but%20not%20the%20%5C%22align%20successive%20assignments%5C%22%20part.%22%2C%20%22created_at%22%3A%20%222016-01-06T10%3A04%3A06Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6430928%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/dawagner%22%7D%7D%2C%20%7B%22body%22%3A%20%22Ok%2C%20%40tcahuzax%20what%20is%20your%20opinion.%22%2C%20%22created_at%22%3A%20%222016-01-06T11%3A05%3A17Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%2C%20%7B%22body%22%3A%20%22Got%20feedback%20from%20team.%22%2C%20%22created_at%22%3A%20%222016-01-11T19%3A56%3A42Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6862950%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/krocard%22%7D%7D%5D%2C%20%22title%22%3A%20%22General%20Comment%22%7D%2C%20%22Pull%20ca70d270e54827a9fb8657355320839e6a051fc5%20bindings/c/ParameterFramework.cpp%2019%22%3A%20%7B%22html_url%22%3A%20%22https%3A//github.com/01org/parameter-framework/pull/327%23discussion_r49092655%22%2C%20%22comments%22%3A%20%5B%7B%22body%22%3A%20%22I%27m%20not%20a%20big%20fan%20of%20this%20alignment%22%2C%20%22created_at%22%3A%20%222016-01-07T16%3A32%3A24Z%22%2C%20%22user%22%3A%20%7B%22avatar_url%22%3A%20%22https%3A//avatars.githubusercontent.com/u/6605131%3Fv%3D3%22%2C%20%22html_url%22%3A%20%22https%3A//github.com/gdenneul%22%7D%7D%5D%2C%20%22title%22%3A%20%22File%3A%20bindings/c/ParameterFramework.cpp%3AL347-354%22%7D%7D%7D'></a>
<a href='https://www.codereviewhub.com/'></a>
- [ ] <a href='#crh-comment-General Comment'></a> &nbsp;<b><a href='https://github.com/01org/parameter-framework/pull/327#issuecomment-169080057'>General Comment</a></b>
- <a href='https://github.com/krocard'></a> @dawagner do you prefer this style ?
- <a href='https://github.com/dawagner'></a> @krocard I like the "binpack" part but not the "align successive assignments" part.
- <a href='https://github.com/krocard'></a> Ok, @tcahuzax what is your opinion.
- <a href='https://github.com/krocard'></a> Got feedback from team.
- [ ] <a href='#crh-comment-Pull ca70d270e54827a9fb8657355320839e6a051fc5 bindings/c/ParameterFramework.cpp 19'></a> &nbsp;<b><a href='https://github.com/01org/parameter-framework/pull/327#discussion_r49092655'>File: bindings/c/ParameterFramework.cpp:L347-354</a></b>
- <a href='https://github.com/gdenneul'></a> I'm not a big fan of this alignment

<a href='https://www.codereviewhub.com/01org/parameter-framework/pull/327?mark_as_completed=1'></a>&nbsp;<a href='https://www.codereviewhub.com/01org/parameter-framework/pull/327?approve=1'></a>&nbsp;<a href='https://github.com/01org/parameter-framework/pull/327'></a>
<a href='#crh-end'></a>

### CODE REVIEW

**Comment by dawagner** [comment]
This pull request is now managed with CodeReviewHub. <a href='https://github.com/01org/parameter-framework/pull/327'>Refresh</a>.<a href='#crh-update-none'></a>

**Comment by krocard** [comment]
@dawagner do you prefer this style ?

**Comment by dawagner** [comment]
@krocard I like the "binpack" part but not the "align successive assignments" part.

**Comment by krocard** [comment]
Ok, @tcahuzax what is your opinion.

File: bindings/c/ParameterFramework.cpp [hunk 1]
```diff
@@ -344,7 +347,7 @@ bool pfwSetIntParameter(PfwParameterHandler *handle, int32_t value)
 bool pfwGetStringParameter(const PfwParameterHandler *handle, char *value[])
 {
     Status &status = handle->pfw.lastStatus;
-    *value = NULL;
+    *value         = NULL;
```
Comment (gdenneul) [inline bindings/c/ParameterFramework.cpp]: I'm not a big fan of this alignment

**Comment by krocard** [comment]
Got feedback from team.

**Comment by krocard** [comment]
We might enable wrap parameter declaration soon.

**Pull request closed without merging**